dist/Constants.d.ts

```diff
@@ -0,0 +1,13 @@
+export declare const SOCKET_PORT = 8000;
```

Comment:
What's the difference between `dist` and `lib` directories?

Reply:
`tsc` build into the `dist` folder right? In that case, I think we should gitignore the directory.

Reply:
Also, let's maybe rename `lib` to `src` since this app isn't a library

Reply:
Good catch.